tests/test_flows.py

```diff
@@ -1246,14 +1249,27 @@ def test_higher_layer_data_payload_trigger(self):
         mid = env.controller.dev_send_1905(env.agents[0].mac, 0x8018,
                                            tlv(0xA0, 0x04b1, "{0x00 %s}" % payload))
 
-        debug("Confirming higher layer data message was received in the agent")
+        debug("Confirming higher layer data message was received in one of the agent's radios")
 
-        self.check_log(env.agents[0], r"HIGHER_LAYER_DATA_MESSAGE")
+        received_in_radio0, _, _ = self.check_log(env.agents[0].radios[0],
```

Comment:
A comment only:
We should stop using check_log to verify correctness.
We can either:
* query the application (best)
* check packets (good)
I leave it as a comment only because this is not a new `check_log`.